tests/Feature/Console/MergeUsersTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace Tests\Feature\Users\Console;
```

Comment:
The file path and namespace are arguing and the file wants to either be in `Feature/Console` or `Feature/Users/Console` 😛